gettext-template/entrypoint.sh

```diff
@@ -35,6 +35,7 @@ echo "Setting up git credentials..."
 git remote set-url origin https://x-access-token:"$GITHUB_TOKEN"@github.com/"$GITHUB_REPOSITORY".git
 git config --global user.email "$GIT_USER_EMAIL"
 git config --global user.name "$GIT_USER_NAME"
+git config --global --add safe.directory /github/workspace
```

Comment:
```suggestion
git config --global --add safe.directory "$GITHUB_WORKSPACE"
```

Does it make more sense to use this environment variable here instead of harcoding this?

Reply:
Actually, I guess not, since this runs in docker. Nevermind